app/forms/work_form.rb

```diff
@@ -7,9 +7,7 @@ class WorkForm < DraftWorkForm
   validates :abstract, :access, :title, presence: true, allow_nil: false
   validates :keywords, length: { minimum: 1, message: 'Please add at least one keyword.' }
   validates :attached_files, length: { minimum: 1, message: 'Please add at least one file.' },
-                             unless: lambda {
-                                       ::ActiveRecord::Type::Boolean.new.cast(globus)
-                                     }
+                             if: -> { upload_type == 'browser' }
   validates :contact_emails, length: { minimum: 1, message: 'Please add at least contact email.' }
```

Comment:
note: this validation will get a bit more complicated once we add option 3 (single ZIP file) in a future PR